sdk/core/loader/boot.S

```diff
@@ -13,6 +13,10 @@
     .p2align 2
     .type start,@function
 start:
+	.rept 32
+	.word 0x0000006f
+	.endr
+
```

Comment:
I've added this to fill the ibex interrupt vector. How should I set up interrupts properly for the CHERIoT RTOS?

Reply:
CHERIoT doesn't use the vectored interrupts (CHERIoT Ibex doesn't support the vectored mode).  Interrupts are configured by the scheduler.  If Sunburst is doing something different for the interrupt controller (not the CLINT + PLIC combination that we normally use with CHERIoT Ibex) then you'll need a custom driver.

Reply:
Ah thanks! I'll have a peruse of the cheriot-safe system to see how the sonata system should be doing this

Reply:
On SAFE, we fill in a bunch of zeroes where the vector should be, but the interrupt vector SCRs are set up by the loader.  This code is shared between the Ibex and Sail targets currently, so hopefully Sonata will do the same thing and not need software changes.